.dockerignore

```diff
@@ -1,3 +1,4 @@
+db/development.sqlite3
```

Comment:
I suggest making this

```
db/*.sqlite3
```

just to avoid picking up any `sqlite3` files, no matter what they are named.

Reply:
Confirmed that you have changed the .dockerignore file